packages/eslint-config/rules/style.js

```diff
@@ -0,0 +1,775 @@
+        ObjectPattern: false,
+        VariableDeclaration: false,
+        NewExpression: false,
+      },
+    }],
+
+
+    /**
+     * enforce consistent spacing inside computed property brackets
+     */
+    'computed-property-spacing': ['error', 'never', { enforceForClassMembers: true }],
+
+
+    /**
+     * enforce consistent naming when capturing the current execution context
+     */
+    'consistent-this': ['error', 'self'],
+
+
+    /**
+     * require or disallow newline at the end of files
+     */
+    'eol-last': ['error', 'always'],
+
+
+    /**
+     * require or disallow spacing between function identifiers and their invocations
+     */
+    'func-call-spacing': ['error', 'never'],
+
+
+    /**
+     * require function names to match the name of the variable or property to which they are assigned
+     */
+    'func-name-matching': ['error', 'always', {
+      considerPropertyDescriptor: true,
+      includeCommonJSModuleExports: true,
+    }],
+
+
+    /**
+     * require or disallow named function expressions
+     */
+    'func-names': ['error', 'always', {
+      generators: 'always',
+    }],
+
+
+    /**
+     * enforce the consistent use of either function declarations or expressions
+     */
+    'func-style': ['error', 'declaration', {
+      allowArrowFunctions: true,
+    }],
+
+
+    /**
+     * enforce line breaks between arguments of a function call
+     */
+    'function-call-argument-newline': ['error', 'consistent'],
+
+
+    /**
+     * enforce consistent line breaks inside function parentheses
+     */
+    'function-paren-newline': ['error', 'consistent'],
+
+
+    /**
+     * disallow specified identifiers
+     */
+    'id-blacklist': ['off'],
+
+
+    /**
+     * enforce minimum and maximum identifier lengths
+     */
+    'id-length': ['error', {
+      min: 2,
+      max: 35,
+    }],
+
+
+    /**
+     * require identifiers to match a specified regular expression
+     */
+    'id-match': ['off'],
+
+
+    /**
+     * enforce the location of arrow function bodies
+     */
+    'implicit-arrow-linebreak': ['error', 'beside'],
+
+
+    /**
+     * enforce consistent indentation
+     */
+    'indent': ['error', 2, {
+      SwitchCase: 1,
+      VariableDeclarator: {
+        var: 2,
+        let: 2,
+        const: 3,
+      },
+      outerIIFEBody: 1,
+      MemberExpression: 1,
+      FunctionDeclaration: {
+        parameters: 1,
+        body: 1,
+      },
+      FunctionExpression: {
+        parameters: 1,
+        body: 1,
+      },
+      CallExpression: {
+        arguments: 1,
+      },
+      ArrayExpression: 1,
+      ObjectExpression: 1,
+      ImportDeclaration: 1,
+      flatTernaryExpressions: true,
+
+      // list derived from https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L143,
+      // which was derived from https://github.com/benjamn/ast-types/blob/HEAD/def/jsx.js,
+      ignoredNodes: [
+        'JSXElement',
```

Comment:
TODO: investigate if this list needs to be updated in any way. it's quite old and things might have changed.